### PR TITLE
Feature/sisu callum dbt20

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
Mirror PR of PR #13 created by @sisu-callum for dbt 20 compatibility

### CODE REVIEW

File: models/marketo.yml
Comment (fivetran-jamie) [inline models/marketo.yml:40]: why are we removing count_sends btw?

File: packages.yml
Comment (fivetran-jamie) [inline packages.yml:4]: reminder to change before merging

**Review by fivetran-jamie — APPROVED** [review]
lgtm after updating the packages.yml

File: packages.yml
Comment (fivetran-joemarkiewicz) [reply packages.yml:4]: Thank youuuuu

File: models/marketo.yml
Comment (fivetran-joemarkiewicz) [reply models/marketo.yml:40]: Good call! @sisu-callum called this out because `persist_docs` was failing his runs because `email_sends` did not exist in the final model and did in the yml. 

@sisu-callum do your runs fail if we only remove the `email_sends` description for the `marketo__email_sends` table? It looks like that is the only table where email_sends is included in the yml and not the final table.

**Review by fivetran-joemarkiewicz — COMMENTED** [review]
One quick question for @sisu-callum

File: models/marketo.yml
Comment (sisu-callum) [reply models/marketo.yml:40]: I removed count_sends because it wasn't being referenced in any of the models. It looks like it might have been a holdover from an older version. 

@fivetran-joemarkiewicz I'm realizing now that we can actually leave this in if we're enforcing `0.20.0` in this package, given the `IF_EXISTS` fix for Snowflake `persist_docs`

**Pull request merged**
